### DISCUSSION TITLE
Gift Patch spawning behavior

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#403] [Q&A] [answered]
The items spawned from gift box still floating when we are using the Patch.
So I wonder, if it's behavior is just like the old **/grab** command, is it possible to change the behavior of the item spawned from gift into our current **/grab?**

**Comment by winstxnhdw** [comment, marked as answer]
It should be possible but I’ll be putting it on the backlog.

**Reply by Dexteriority** [reply]
Thank you!!
